python/cudf/cudf/_lib/datetime.pyx

```diff
@@ -61,29 +61,54 @@ def extract_datetime_component(Column col, object field):
 
     return result
 
-
-def ceil_datetime(Column col, object field):
+#cdef _get_rounding_frequency(object freq):
+#    cdef libcudf_datetime.rounding_frequency freq_val
+#
+#    # https://pandas.pydata.org/pandas-docs/version/0.25.0/reference/api/pandas.Timedelta.resolution.html
+#    if freq == "D":
+#        freq_val = libcudf_datetime.rounding_frequency.DAY
+#    elif freq == "H":
+#        freq_val = libcudf_datetime.rounding_frequency.HOUR
+#    elif freq == "T" or freq == "min":
+#        freq_val = libcudf_datetime.rounding_frequency.MINUTE
+#    elif freq == "S":
+#        freq_val = libcudf_datetime.rounding_frequency.SECOND
+#    elif freq == "L" or freq == "ms":
+#        freq_val = libcudf_datetime.rounding_frequency.MILLISECOND
+#    elif freq == "U" or freq == "us":
+#        freq_val = libcudf_datetime.rounding_frequency.MICROSECOND
+#    elif freq == "N":
+#        freq_val = libcudf_datetime.rounding_frequency.NANOSECOND
+#    else:
+#        raise ValueError(f"Invalid resolution: '{freq}'")
+#    return freq_val
+
+def ceil_datetime(Column col, object freq):
     cdef unique_ptr[column] c_result
     cdef column_view col_view = col.view()
+    #cdef libcudf_datetime.rounding_frequency freq_val = _get_rounding_frequency(freq)
+
+    #with nogil:
+    #    c_result = move(libcudf_datetime.ceil_datetimes(col_view, freq_val))
```

Comment:
TODO: This path needs to be uncommented and the other section below needs to be removed.